src/style.css

```diff
@@ -1 +1,35 @@
 /* styles go here */
+.form-field {
+  margin-bottom: 20px;
+  padding-bottom: 4px;
```

Comment:
[CHECKLIST](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) - 10
![image](https://user-images.githubusercontent.com/86923089/219941031-a1a175e4-90ae-43ed-b4c3-c6cf074c9320.png)

no need to set paddings
you may get the same result with simpler code

use only bottom or only top margins - that's a good practice (at least you'll avoid margin collapse)

Reply:
If you need help, or have questions - feel free to ask in chat

Reply:
I can't using only margin-top or only margin-bottom. I don't know how it worked separately. And I don't know how not use ".label {
  display: block;
}"